### PR TITLE
frontend: remove codemirror file preview modal

### LINKED ISSUE #2997: frontend: fix UI according to new webpack build system
1. Upgrade last Invenio UI packages:
   - [x] Upgrade to latest Invenio-Search-UI (https://github.com/cernopendata/opendata.cern.ch/pull/3015 @mvidalgarcia)
   - [x] Upgrade to latest Invenio-Records-UI (https://github.com/cernopendata/opendata.cern.ch/pull/2998 @diegodelemos)
   - [x] Upgrade to latest Invenio-Previewer (https://github.com/cernopendata/opendata.cern.ch/pull/3002 @audrium)

2. Upgrade bundles to new webpack build system (documentation [here](https://invenio-assets.readthedocs.io/en/latest/upgrade.html))
   - [x] [Global CSS bundle](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L48) (https://github.com/cernopendata/opendata.cern.ch/pull/3000 @diegodelemos)
   - [x] [Glossary bundle CSS and JS](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L112-L125) (@diegodelemos)
   - [x] [Front JS bundle (not working, do we want it?)](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L68) (@mvidalgarcia) - Removed from now
   - [x] [Visualise bundle](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L91-L110)
   - [x] [Opera viewer](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L127-L168)
   - [x] [ispy bundle](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L170-L223)
   - [x] [Codemirror bundle](https://github.com/cernopendata/opendata.cern.ch/blob/76abc14f0f49f58a14a9c3b792c6daaeab67824a/cernopendata/modules/theme/bundles.py#L227-L251)

### PR DESCRIPTION
**Pull request by diegodelemos**
Closes #2997

Example record: http://opendata.cern.ch/record/7727#

### CODE REVIEW

**Comment by mvidalgarcia** [comment]
@diegodelemos Should be ready to be reviewed.

There are alternatives to implement code highlighting such as [`highlight.js`](https://github.com/highlightjs/highlight.js), but one has to specify the language so it'll be hard to identify it to set the proper class. I think this should be part of a different task as MVP is achieved with this PR.

File: cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss [hunk 1]
```diff
@@ -97,6 +97,12 @@ body {
     }
 }
 
+.config-file-preview-modal {
+    .config-content {
+        white-space: pre-wrap;
```
Comment (diegodelemos) [inline cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss:102]: Moderate: I think that we need to add `overflow: auto` so we have horizontal scrolling, a few records fit by default but some like http://0.0.0.0:5000/record/7727 would require horizontal scrolling.

**Comment by diegodelemos** [comment]
This is ready and neat (can't approve), just the horizontal scrolling is missing.

File: cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss (hunk 1 above)
Comment (mvidalgarcia) [reply cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss:102]: Have you run webpack? I can't reproduce it.

<details>
<summary>Screenshots of record 7727</summary>
<div>

image

image

image

</div>
</details>

Comment (diegodelemos) [reply cernopendata/modules/theme/assets/semantic-ui/scss/styles.scss:102]: I did, `cernopendata webpack clean buildall`, but still somehow text was coming out of the modal. However, after rebuilding the image it worked, strange. Ready to be merged!

**Review by mvidalgarcia — APPROVED** [review]

**Pull request merged**